ttnn/cpp/ttnn/operations/creation.hpp

```diff
@@ -189,7 +189,28 @@ struct FullLikeWith {
     }
 };
 
-struct ZerosLike : FullLikeWith<0.0f> {};
+struct ZerosLike {
```

Comment:
these changes should be made in `struct FullLikeWith ` so that they apply to all the inherited structs as well

Reply:
Here, all of zeroslike, oneslike and emptylike use the same struct. But the issue here is that all of these take in the only same tensor whereas the rest are optional. So there is an issue there. So to avoid confusion, I've created a separate struct for this